### DISCUSSION TITLE
when is next release for CVE-2020-15228 ?

### DISCUSSION THREAD

**Discussion by tooptoop4** [aio-libs/aiohttp#6827] [Q&A]
when is next release for CVE-2020-15228 ?

**Comment by Dreamsorcerer** [comment]
What does that CVE have to do with aiohttp?

**Reply by tooptoop4** [reply]
https://github.com/aio-libs/aiohttp/blob/3a41da05bcb1edde9f6f878d9092e108e70dff2a/Makefile#L64

**Reply by tooptoop4** [reply]
![image](https://user-images.githubusercontent.com/33283496/185056886-78e8cd14-c049-4599-94a3-9f1741a49b9b.png)

**Reply by Dreamsorcerer** [reply]
Still not clear how that affects aiohttp. It certainly doesn't look like something that affects users, maybe affects our CICD pipeline or something.

**Reply by Dreamsorcerer** [reply]
But, it looks like we would need to update llhttp first. So, making a new aiohttp release won't resolve this until that is done.
Feel free to make a PR to update it.
